src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -181,12 +204,22 @@ const GeneExternalReferences = () => {
   );
 };
 
-const RenderTranscriptXrefGroup = (props: { transcript: Transcript }) => {
+const RenderTranscriptXrefs = (props: { transcript: Transcript }) => {
   const { transcript } = props;
   const [isExpanded, setIsExpanded] = useState(false);
-  const transcriptsXrefGroups = buildExternalReferencesGroups(
-    transcript.external_references
+  const unsortedXrefs = [...transcript.external_references];
+
+  // Add protein level xrefs
+  transcript.product_generating_contexts.forEach(
+    (product_generating_context) => {
+      return unsortedXrefs.push(
```

Comment:
Do we need `return` inside a `forEach` function?

Reply:
No, we don't but it won't make any difference. I have removed it now.